vendor/vendor.json

```diff
@@ -59,6 +59,12 @@
 			"revisionTime": "2016-09-30T00:14:02Z"
 		},
 		{
+			"checksumSHA1": "DYv6Q1+VfnUVxMwvk5IshAClLvw=",
```

Comment:
Huh weird, I wonder why it wasn't in here (with the old verison) before. Someone must've added it without govendor or something.